download/save.go

```diff
@@ -0,0 +1,30 @@
+package download
+
+import "os"
+import "github.com/pkg/errors"
+import "io"
+
+const (
+	writeBufSize = 1024 * 2014
```

Comment:
Hmm, I could use advice. I was planning to use io.CopyBuffer and let go decide the buffer size. It appears like the old code has 1 KB of buffer size.